### ISSUE TITLE
contig lines malformed error

### ISSUE THREAD

**Issue by deannachurch** [jamescasbon/PyVCF#200]
Hi,
I'm trying to parse the latest ClinVar VCF and run into this error:

SyntaxError: One of the contig lines is malformed: ##contig=<ID=1>

The lines in the ClinVar VCF look like:
````
##contig=<ID=1>
##contig=<ID=2>
##contig=<ID=3>
##contig=<ID=4>
##contig=<ID=5>
##contig=<ID=6>
##contig=<ID=7>
##contig=<ID=8>
##contig=<ID=9>
##contig=<ID=10>
##contig=<ID=11>
##contig=<ID=12>
##contig=<ID=13>
##contig=<ID=14>
##contig=<ID=15>
##contig=<ID=16>
##contig=<ID=17>
##contig=<ID=18>
##contig=<ID=19>
##contig=<ID=20>
##contig=<ID=21>
##contig=<ID=22>
##contig=<ID=X>
##contig=<ID=Y>
##contig=<ID=MT>
````

Not great, but it seems unfortunate this kills off the whole thing. Can this just be a warning instead of killing off the whole process?

**Comment by deannachurch** [comment]
Note: if I remove those lines, the vcf is read just fine...

**Comment by mattions** [comment]
+1 I've encountered this as well

**Comment by martijnvermaat** [comment]
This has been fixed in master since the last release.

@jamescasbon Can we do a new release? There have been quite some fixes. The only potential problem I see is with pysam, there have been some incompatibility issues when it moved to 0.8.0 and I'm not sure where that stands at the moment. I'd have to double check we're not regressing on that point.

**Comment by deannachurch** [comment]
Is there any update on this? We'd like to be able to process the ClinVar VCF without it hanging for this reason.

**Comment by martijnvermaat** [comment]
@deannachurch While waiting for a proper release with the fix on the Python Package Index, you can install the development version of PyVCF with pip like this:

    pip install -e git+https://github.com/jamescasbon/PyVCF.git#egg=PyVCF

I hope this is an acceptible workaround for you while we prepare a new release.

**Comment by deannachurch** [comment]
Hi Martin-
Do you have a time frame for the new release? That knowledge will make it easier to know if we need to download the dev version or if we can wait for the full release. 
thanks so much for this great package!

**Comment by martijnvermaat** [comment]
@jamescasbon owns the PyPI account and I think he doesn't have much time to work on PyVCF. I'm sorry to keep you waiting.

**Comment by deannachurch** [comment]
Hi Martin,
Thanks for the info.

**Comment by mpschr** [comment]
Hi @martijnvermaat and @jamescasbon 

I've got the same problem and when i do the install via the suggested command above (`pip install -e git+https://github.com/jamescasbon/PyVCF.git#egg=PyVCF`),  the version `PyVCF-0.6.7` is installed which has this very problem. How would I install a fixed version?

**Comment by mattions** [comment]
@mpschr The version installed should be master, however  you can not trust the version on the __init__, because it was not being updated to development mode after the release (see #203 )

Master should be able to read these file, 'cause the header parsing has been relaxed

**Comment by mpschr** [comment]
HI @mattions, Thanks for your answer. I am not quite following you. What is it I cannot trust, the version number when I install from GitHub?

In any case - installing from the github-repository with the mentioned command, does not solve the issue. Do you know what the issue may be?

**Comment by mattions** [comment]
Hi @mpschr .
What I mean is that the current development version of pyVCF (master) has the 0.6.7 version, because it was not changed after the release as you can see from this line: https://github.com/jamescasbon/PyVCF/blob/master/vcf/__init__.py#L193 

To make that a bit more clear, I've made a super simple PR #203, so the version gets bumped in development mode.

if you uninstall completely pyvcf, and than install it via pip you should get master. You can double check this if you look at the code and have the latest master (for example this is the last commit: https://github.com/jamescasbon/PyVCF/commit/ec193b12b9443dc40b0ea49cde134fe60915b60f)

Once you know you have the latest master working you can try to load the file, and than it should work. If it doesn't, than also latest master can't load the file in question.

Hope it's a bit more clear.

**Comment by jsmedmar** [comment]
Any update on this?

**Comment by martijnvermaat** [comment]
This fix is in release 0.6.8 available from PyPI.

Please open a new issue if your problem is not fixed with this.

**Comment by jsmedmar** [comment]
Ok @martijnvermaat, I opened #234.
